packages/@ember/-internals/glimmer/tests/integration/components/link-to/routing-curly-test.js

```diff
@@ -1822,44 +1822,6 @@ moduleFor(
       assertNav({ prevented: true }, () => this.$('#self-link > a').click(), assert);
     }
 
-    async [`@test [DEPRECATED] it should not transition if target is not equal to _self or empty`](
-      assert
-    ) {
-      this.addTemplate(
-        'index',
-        `
-        <div id='about-link'>
-          {{#link-to route='about' replace=true target='_blank'}}
-            About
-          {{/link-to}}
-        </div>
-        `
-      );
-
-      this.router.map(function () {
-        this.route('about');
-      });
-
-      await expectDeprecationAsync(
-        () => this.visit('/'),
-        /Passing the `@target` argument to <LinkTo> is deprecated\./,
-        EMBER_MODERNIZED_BUILT_IN_COMPONENTS
-      );
-
-      await this.click('#about-link > a');
-
-      expectDeprecation(() => {
-        let currentRouteName = this.applicationInstance
-          .lookup('controller:application')
```

Comment:
Again, I this test should be retained with the other `expectDeprecation` left in place, but the code for the specific deprecated API removed here refactored.